workflow/controller/inline_test.go

```diff
@@ -74,3 +75,160 @@ spec:
 	assert.Equal(t, "message", node.Inputs.Parameters[0].Name)
 	assert.Equal(t, "foo", node.Inputs.Parameters[0].Value.String())
 }
+
+var workflowCallTemplateWithInline = `
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  name: test-call-inline-iterated
+  namespace: argo
+spec:
+  entrypoint: main
+  templates:
+    - name: main
+      dag:
+        tasks:
+          - name: process
+            templateRef:
+              name: test-inline-iterated
+              template: main`
+
+var workflowTemplateWithInlineSteps = `
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: test-inline-iterated
+  namespace: argo
+spec:
+  entrypoint: main
+  templates:
+    - name: main
+      steps:
+        - - name: iterated
+            template: steps-inline
+            arguments:
+              parameters:
+                - name: arg
+                  value: "{{ item }}"
+            withItems:
+              - foo
+              - bar
+
+    - name: steps-inline
+      inputs:
+        parameters:
+          - name: arg
+      steps:
+        - - name: inline
+            arguments:
+              parameters:
+                - name: arg
+                  value: "{{ inputs.parameters.arg }}"
+            inline:
+              inputs:
+                parameters:
+                  - name: arg
+              container:
+                image: docker/whalesay
+                command: [echo]
+                args:
+                  - "{{ inputs.parameters.arg }}"
+              outputs:
+                parameters:
+                  - name: arg-out
+                    value: "{{ inputs.parameters.arg }}"
+`
+
+func TestCallTemplateWithInlineSteps(t *testing.T) {
```

Comment:
Given the code needs to successfully find multiple steps in a template could you add a second step with an inline template here too to prove that works and continues to work.

Reply:
OK, I will, thanks